components/collections/TokenCard.tsx

```diff
@@ -90,6 +92,26 @@ export default ({
           <Text ellipsify>x{tokenCount}</Text>
         </Flex>
       )}
+      {orderQuantity && orderQuantity > 1 && (
```

Comment:
Set to only render if the order quantity is greater than 1 so that not every single token has a badge when it's just a single order.